go.mod

```diff
@@ -0,0 +1,170 @@
+	github.com/coreos/ignition v0.34.0
+	github.com/dmacvicar/terraform-provider-libvirt v0.6.1-0.20191216113711-1a01934a7d20
+	github.com/frankban/quicktest v1.7.2 // indirect
+	github.com/gammazero/workerpool v0.0.0-20191005194639-971bc780f6d7 // indirect
+	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
+	github.com/go-logr/zapr v0.1.1 // indirect
+	github.com/golang/mock v1.3.1
+	github.com/google/go-cmp v0.3.2-0.20191028172631-481baca67f93 // indirect
+	github.com/google/martian v2.1.1-0.20190517191504-25dcb96d9e51+incompatible // indirect
+	github.com/google/uuid v1.1.1
+	github.com/gophercloud/gophercloud v0.7.1-0.20191210042042-7aa2e52d21f9
+	github.com/gophercloud/utils v0.0.0-20191129022341-463e26ffa30d
+	github.com/gopherjs/gopherjs v0.0.0-20191106031601-ce3c9ade29de // indirect
+	github.com/gorilla/websocket v1.4.1 // indirect
+	github.com/grpc-ecosystem/grpc-gateway v1.12.1 // indirect
+	github.com/hashicorp/go-azure-helpers v0.10.0
+	github.com/hashicorp/go-plugin v1.0.1
+	github.com/hashicorp/go-retryablehttp v0.6.4 // indirect
+	github.com/hashicorp/logutils v1.0.0
+	github.com/hashicorp/terraform v0.12.18
+	github.com/hashicorp/terraform-plugin-sdk v1.4.1
+	github.com/hashicorp/vault v1.3.0 // indirect
+	github.com/hinshun/vt10x v0.0.0-20180809195222-d55458df857c // indirect
+	github.com/joyent/triton-go v1.7.0 // indirect
+	github.com/keybase/go-crypto v0.0.0-20190828182435-a05457805304 // indirect
+	github.com/kubernetes-sigs/aws-iam-authenticator v0.4.0 // indirect
+	github.com/libvirt/libvirt-go v5.10.0+incompatible
+	github.com/libvirt/libvirt-go-xml v5.10.0+incompatible // indirect
+	github.com/lusis/go-artifactory v0.0.0-20180304164534-a47f63f234b2 // indirect
+	github.com/masterzen/winrm v0.0.0-20190308153735-1d17eaf15943 // indirect
+	github.com/metal3-io/baremetal-operator v0.0.0
+	github.com/metal3-io/cluster-api-provider-baremetal v0.0.0
+	github.com/mitchellh/cli v1.0.0
+	github.com/onsi/ginkgo v1.10.3 // indirect
+	github.com/onsi/gomega v1.7.1 // indirect
+	github.com/opencontainers/image-spec v1.0.2-0.20190823105129-775207bd45b6 // indirect
+	github.com/openshift-metal3/terraform-provider-ironic v0.1.9
+	github.com/openshift/api v3.9.1-0.20191111211345-a27ff30ebf09+incompatible
+	github.com/openshift/client-go v0.0.0-20191001081553-3b0e988f8cb0
+	github.com/openshift/cloud-credential-operator v0.0.0-20191205223904-83cfad33adf5
+	github.com/openshift/cluster-api v0.0.0-20191129101638-b09907ac6668
+	github.com/openshift/cluster-api-provider-gcp v0.0.0-20190826205919-0cd5daa07e0d
+	github.com/openshift/cluster-api-provider-libvirt v0.0.0-20190613141010-ecea5317a4ab
+	github.com/openshift/library-go v0.0.0-20191205152556-73e1fb871a9b
+	github.com/openshift/machine-api-operator v0.2.1-0.20200116114648-a3345f5a437a
+	github.com/openshift/machine-config-operator v4.2.0-alpha.0.0.20190917115525-033375cbe820+incompatible
+	github.com/ovirt/cluster-api-provider-ovirt v0.0.0
```

Comment:
now I understand - please use `github.com/openshift/cluster-api-provider-ovirt`

Reply:
I can only do that once that module starts declaring its name accordingly, i.e.  https://github.com/openshift/cluster-api-provider-ovirt/pull/2 goes in

Reply:
btw this is replaced by github.com/openshift/cluster-api-provider-ovirt below - so the version actually imported is the openshift one.

Reply:
I fixed the module path in  https://github.com/openshift/cluster-api-provider-ovirt/pull/3

Reply:
thank you! :) I'll update this PR shortly